tests/messages/test_views.py

```diff
@@ -34,36 +34,38 @@ def subscription_data():
     dataset = DatasetFactory()
     project = ProjectFactory()
     user = UserFactory(organization=org1)
-    return {
-        'request': request,
-        'dataset': dataset,
-        'project': project,
-        'user': user
-    }
+    return {"request": request, "dataset": dataset, "project": project, "user": user}
 
 
 @pytest.mark.django_db
 def test_request_subscribe_without_user(app: DjangoTestApp, subscription_data):
-    resp = app.get(subscription_data['request'].get_absolute_url())
+    resp = app.get(subscription_data["request"].get_absolute_url())
     assert Subscription.objects.count() == 0
 
-    elem = resp.html.find(id='number-of-subscribers')
-    assert elem.get_text().strip() == '0'
+    elem = resp.html.find(id="number-of-subscribers")
+    assert elem.get_text().strip() == "0"
 
 
 @pytest.mark.django_db
 def test_request_subscribe_url_no_login(app: DjangoTestApp, subscription_data):
-    resp = app.get(subscription_data['request'].get_absolute_url())
-    elem = resp.html.find(id='request_subscription')
-    form_url = elem.find('a', {'id': 'subscribe-form'})
+    resp = app.get(subscription_data["request"].get_absolute_url())
```

Comment:
Let's avoid running formatting together w/ the functional changes, or at the least, make multiple commits (one for functional, other one for formatting) and mention them in the PR description. It is quite hard to review the PR, when these changes intertwine